frontend/src/shared/modules/filters/components/filterTypes/DateFilter.vue

```diff
@@ -65,16 +62,18 @@ const rules: any = {
   },
 };
 
+const checkIfBetween = (operator) => [FilterDateOperator.BETWEEN, FilterDateOperator.NOT_BETWEEN].includes(operator);
```

Comment:
Should this be a computed property?

Reply:
I compare previous value with current so i added now also computed property for faster rendering